MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -275,14 +280,12 @@ public void processGame() {
                         status.assignFoundEntity(e, true);
                     }
                 }
+            }
+            Entity killer = victoryEvent.getEntity(e.getKillerId());
+            if (null != killer && !killer.getExternalIdAsString().equals("-1")) {
```

Comment:
brackets, inverse the "-1" equals to prevent null issues (safety change).

Reply:
I am not sure what inverse means in this context.

Reply:
!"-1".equals(killer.getExternalIdAsString())

Unlike null leading checks this is rarely handled by IDEs and causes minimal processing delays